pkg/nsx/services/securitypolicy/firewall.go

```diff
@@ -428,10 +429,17 @@ func (service *SecurityPolicyService) convertNetworkPolicyPeerToSecurityPolicyPe
 }
 
 func (service *SecurityPolicyService) convertNetworkPolicyPortToSecurityPolicyPort(npPort *networkingv1.NetworkPolicyPort) (*v1alpha1.SecurityPolicyPort, error) {
-	spPort := &v1alpha1.SecurityPolicyPort{
-		Protocol: *npPort.Protocol,
-		Port:     *npPort.Port,
+	spPort := &v1alpha1.SecurityPolicyPort{}
+	if npPort.Protocol != nil {
+		spPort.Protocol = *npPort.Protocol
 	}
+
+	if npPort.Port != nil {
+		spPort.Port = *npPort.Port
+	} else {
+		spPort.Port = intstr.FromString("ANY")
```

Comment:
I don't think we handle Port with ANY value.  
As usauulay, we set a port range in nsx destination port. If we want to set ANY in destination port, we need to have some handling in buildRuleServiceEntries